FBTweak/FBTweakCollection.h

```diff
@@ -12,42 +12,48 @@
 @protocol FBTweak;
 
 /**
-  @abstract A named collection of tweaks.
+  @abstract A named collection of Tweaks.
  */
 @interface FBTweakCollection : NSObject <NSCoding>
 
+- (instancetype)init NS_UNAVAILABLE;
+
 /**
-  @abstract Creates a tweak collection.
-  @discussion This is the designated initializer.
+  @abstract Creates an empty Tweak collection.
  */
 - (instancetype)initWithName:(NSString *)name;
 
+/**
+ @abstract Creates a tweak category with an initial Tweak array.
+ */
+- (instancetype)initWithName:(NSString *)name tweaks:(NSArray<id<FBTweak>> *)tweaks NS_DESIGNATED_INITIALIZER;
+
 /**
   @abstract The name of the collection.
  */
 @property (nonatomic, copy, readonly) NSString *name;
 
 /**
-  @abstract The tweaks contained in this collection.
+  @abstract The Tweaks contained in this collection.
  */
-@property (nonatomic, copy, readonly) NSArray *tweaks;
+@property (nonatomic, copy, readonly) NSArray<id<FBTweak>> *tweaks;
```

Comment:
What the `copy` is good for?
IMO it can be omitted since the getter is implemented as follows:
```objc
 - (NSArray<id<FBTweak>> *)tweaks
  {
    return [_orderedTweaks copy];
  }
```

Right?

Reply:
`copy` is only useful in writable properties and does nothing here. I did an overhaul on the class fixing this and making it all LT like style.